hack/remove-calico-policy/README.md

```diff
@@ -1,4 +1,41 @@
-# Disabling and Removing Calico Policy
+# Disabling Calico Policy
+
+This guide explains two methods for disabling Calico policy on a running cluster. These steps are intended as a
+last resort should it appear that Calico policy is not functioning properly on a production system.
+
+- [Overriding Calico Policy] describes how to add high-priority Calico System Network Policy which
+  overrides the configured Network Policy.
+- [Disabling and Removing Calico Policy] describes how to fully disable Calico policy (removing all 
+  Calico-programmed policy from each node.
+
+## Overriding Calico Policy
```

Comment:
Let's move this to a separate markdown file, and maybe just have a link from this doc along the lines of "These steps are intended as a last resort.  You probably want to try this first"